.github/workflows/pr-change-set.yaml

```diff
@@ -18,3 +18,23 @@ jobs:
     uses: ./.github/workflows/build.yaml
     permissions:
       contents: read
+
+  comment-audit:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    needs: build
+    steps:
+
+    - name: Download artifact from Build
+      uses: actions/download-artifact@9782bd6a9848b53b110e712e20e42d89988822b7 #v3
+      if: steps.audit-packages.outputs.exit_code == 1
+      with:
+        name: vulnerabilities.txt
+
+    - name: comment PR
```

Comment:
Title case, for consistency: `Comment on PR`

Reply:
Adjusted the name!